pkg/pipeline/CiHandler.go

```diff
@@ -1303,23 +1303,28 @@ func (impl *CiHandlerImpl) FetchCiStatusForTriggerViewForEnvironment(envId int,
 		impl.Logger.Warnw("there is no app id found for fetching ci pipelines", "envId", envId)
 		return ciWorkflowStatuses, nil
 	}
-	pipelines, err := impl.ciPipelineRepository.FindByAppIds(appIds)
+	ciPipelines, err := impl.ciPipelineRepository.FindByAppIds(appIds)
 	if err != nil && err != pg.ErrNoRows {
 		impl.Logger.Errorw("error in fetching ci pipeline", "err", err)
 		return ciWorkflowStatuses, err
 	}
-
+	pipelineIds := make([]int, 0)
```

Comment:
this should be ciPipelineIds

Reply:
done

Reply:
I have already pulled out appIds based on cdPipeline, if I check permission on the apps and then fetch ciPipelines for only those apps then it might be faster and reduce some db calls